composer.json

```diff
@@ -25,7 +25,7 @@
         "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^3.3",
+        "friendsofphp/php-cs-fixer": "~3.4.0",
```

Comment:
I don't think we should lock the version of `php-cs-fixer`, maybe we just adjust the rules and code as they update?